app/views/historic_appointments/show.html.erb

```diff
@@ -1,57 +1,74 @@
-      url: "/government/history"
-    }
-  ]
-} %>
-<header class="block headings-block">
-  <div class="govuk-width-container">
-    <%= render "govuk_publishing_components/components/title", {
-      context: {
-        text: "History",
-      },
-      title: "Past #{@role.name.pluralize}",
+<% page_class "govuk-width-container" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/breadcrumbs", {
+      breadcrumbs: [
+        {
+          title: "Home",
+          url: "/"
+        },    
+        {
+          title: "History of the UK Government",
+          url: "/government/history"
+        }
+      ]
     } %>
   </div>
-</header>
 
-<%= render partial: 'role_appointments_list', locals: { role: @role } %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/title", {
+      context: "Past #{@role.name.pluralize}",
+      title: @person.name,
+    } %>
 
-<%= content_tag_for(:div, @historical_account, class: "block person-info") do %>
-  <div class="name-title">
-    <div class="inner">
-      <h2>
-        <%= @person.name %>
-        <span><%= @historical_account.political_membership %> <%= previous_dates_in_office(@role, @person) %></span>
-      </h2>
-    </div>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: "#{@historical_account.political_membership} #{@historical_account.previous_dates_in_office}",
+    } %>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: @historical_account.summary,
+    } %>
   </div>
-  <div class="person-profile">
-    <div class="inner">
-      <div class="info">
-        <%= @person.image %>
-        <%= historical_fact('Born', @historical_account.born) %>
-        <%= historical_fact('Died', @historical_account.died) %>
-        <%= historical_fact('Dates in office', previous_dates_in_office(@role, @person)) %>
-        <%= historical_fact('Political party', @historical_account.political_membership) %>
-        <%= historical_fact('Major acts', format_with_html_line_breaks(@historical_account.major_acts)) %>
-        <%= historical_fact('Interesting facts', format_with_html_line_breaks(@historical_account.interesting_facts)) %>
+
+  <div class="govuk-grid-column-two-thirds">
+    <div class="person-profile">
+      <div class="person-profile__image">
+        <%= image_tag @person.image_url, alt: "", loading: "lazy" %>
+      </div>
+      <div class="person-profile__text">
+        Picture of <%= @person.name %>
       </div>
     </div>
+
+    <% @historical_account.appointment_info_array.each do |info| %>
+      <% unless info[:text].blank? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: info[:title],
+          font_size: "m",
+          margin_bottom: 1,
+        } %>
+        <p class="govuk-body"><%= info[:text] %></p>
```

Comment:
👀

Reply:
What do these eyes mean?

Reply:
Vague emoji use 😁  - at the time of selecting I believe they meant: I like this swap technique - let me take a closer look.